File: 프로그래머스/lv0/120852. 소인수분해/소인수분해.py
def solution(n):
    #n이 나누어떨어지는 소수들을 구하면 됨
    answer = []
    for i in range(2,n+1):
        count = 0
        for j in range(1,i+1):
            if i%j==0:
                count+=1
        if count==2 and n%i==0:
            answer.append(i)
    return sorted(answer)

File: 프로그래머스/lv0/120852. 소인수분해/소인수분해.js
function solution(n) {
    var answer = [];
    let a = 2;
    while (n>1){
        while (n%a===0){
            if (!answer.includes(a)){
                answer.push(a);
            }
            n=n/a;
        }
        a++;
    }
    return answer;
}